src/app/page/Request/foodbox/FoodboxFlow.jsx

```diff
@@ -34,66 +26,39 @@ const useStyles = makeStyles((theme) => ({
 const stepComponents = [FoodboxStep, DeliveryStep, ConfirmStep];
 const tabNames = ["FOODBOX", "DELIVERY", "CONFIRM"];
 
-const defaultFormValues = {
-  quantity: 1,
-  basket: mockData.BASKET_NAME,
-};
-
 export default function FoodboxFlow() {
   const classes = useStyles();
-  const { handleChange, values } = useForm(defaultFormValues);
-
-  const [activeStep, setActiveStep] = useState(0);
-  const [loading, setLoading] = useState(false);
-  const [errorSnackbarMessage, setErrorSnackbarMessage] = useState(false);
-
-  const handleNext = () => {
-    setActiveStep((prevActiveStep) => prevActiveStep + 1);
-  };
-
-  const handleBack = () => {
-    setActiveStep((prevActiveStep) => prevActiveStep - 1);
-  };
-
-  const handleTabClick = (newStep) => {
-    setActiveStep(newStep);
-  };
+  const [state, dispatch] = useReducer(reducer, initialState);
+  console.log(state);
 
-  const ActiveComponent = stepComponents[activeStep];
+  const ActiveComponent = stepComponents[state.step];
 
-  if (loading) return <CircularProgress />;
+  if (state.loading) return <CircularProgress />;
 
   return (
     <div className={classes.root}>
```

Comment:
we have a hoc withLoading

Reply:
Looking back at this I think I'll just remove this for now.